spec/API_specification/array_object.md

```diff
@@ -434,7 +434,13 @@ Exports the array for consumption by {ref}`function-from_dlpack` as a DLPack cap
 
 -   **stream**: _Optional\[ int ]_
```

Comment:
```suggestion
-   **stream**: _Any_
```
You could also go for `Optional[Union[int, Any]]` or `Union[int, None, Any]`, but in the end all these statements are equivalent.

Reply:
Thanks Bas, I missed updating that. From a typing point of view they're indeed all the same. From a "this serves as documentation as well" point of view, I think I prefer `Optional[Union[int, Any]]`.